code/obj/item/pens_writing_etc.dm

```diff
@@ -1118,53 +1118,3 @@
 		else
 			..()
 		return
-
-/* =============== STICKY NOTES =============== */
-
-/obj/item/postit_stack
```

Comment:
For now please don't remove the definition itself, as it might be slightly more headache for finding it in other maps that haven't been updated in a hot minute. 

Suggestion: Keep the def, change name and desc to "you shouldnt see this" or something of that nature, and comment-out the rest, we'll housekeep it later I think.

Reply:
( i am of course talking about SECRET CONTENT and if you don't do this im going to shit and piss my pants like a jerk)

Reply:
done